src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorageWithProperties/AzureTableStorageEntityFactory.cs

```diff
@@ -61,9 +61,15 @@ public static DynamicTableEntity CreateEntityWithProperties(LogEvent logEvent, I
 
             List<KeyValuePair<ScalarValue, LogEventPropertyValue>> additionalData = null;
             var count = dynamicProperties.Count;
+			bool isNumeric;
+			int parsedNumber;
+
             foreach (var logProperty in logEvent.Properties)
             {
-                if (count++ < _maxNumberOfPropertiesPerRow - 1)
+				isNumeric = int.TryParse(logProperty.Key, out parsedNumber);
```

Comment:
Can we encapsulate this check in a function `IsValidColumnName(s)`? Although the check for numerics is an improvement, Serilog allows (e.g. through enrichers) all kinds of property names, for example `hello-world` or `a.b:c`. In the future we'll need to handle these cases properly, too.